### ISSUE TITLE
[Rebrand] Update WooPay shopper feedback survey

### ISSUE THREAD

**Issue by leonardola** [Automattic/woocommerce-payments#9807]
9Mk09c2GvJBVRW6hwjlOFo-fi-8_16224

Should look like this:

![Image](https://github.com/user-attachments/assets/57b31459-f065-4de2-9fe2-f58671b8d4a2)

@pierorocca  I believe you'll need to update this one right?

**Comment by pierorocca** [comment]
> @pierorocca I believe you'll need to update this one right?

Yes correct. Changes made will apply real-time so I think i'd need to defer this change to near the launch date @elizaan36?

**Comment by elizaan36** [comment]
Yes, exactly. The launch date is Feb 4th!

**Comment by deepakpathania** [comment]
@pierorocca Could you confirm if this is still needed?

**Comment by pierorocca** [comment]
@deepakpathania I'll close it. It was updated and I have a different issue open to migrate the survey from Airtable to Typeform.
